ResearchKit/1.4.1/ResearchKit.podspec

```diff
@@ -1,15 +0,0 @@
-Pod::Spec.new do |s|
```

Comment:
Is removing this file intensional? I don't use cocoa-pods and don't really remember.